File: Dokany UI/src/app/models/order-wizard/ShippingAddress.ts
export class ShippingAddress{
    id: Number;
    firstName: String;
    lastName: String;
    addressLine1: String;
    addressLine2: String;
    zipCode: String;
    city: String;
    country: String;
    state: String;
    phoneNumber: String;

    constructor(
        Id: Number , 
        FirstName: String , 
        LastName: String , 
        PhoneNumber: String,
        AddressLine1: String, 
        AddressLine2: String, 
        ZipCode: String = '', 
        City: String ='',
        Country: String = '',
        State: String = ''
        ){
        this.id = Id;
        this.firstName = FirstName;
        this.lastName = LastName;
        this.addressLine1 = AddressLine1;
        this.addressLine2 = AddressLine2;
        this.zipCode = ZipCode;
        this.city = City;
        this.country = Country;
        this.state = State;
        this.phoneNumber = PhoneNumber;
    }
}

File: Dokany.App/DokanyApp.BLL/Entities/ShippingAddress.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace DokanyApp.BLL
{
    public class ShippingAddress
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string ZipCode { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public string State { get; set; }
        public string PhoneNumber { get; set; }
        [ForeignKey(nameof(User))]
        public int UserId { get; set; }
        public User User { get; set; }

    }
}
